src/app/components/FrostedGlassPromo/index.test.tsx

```diff
@@ -86,6 +87,7 @@ describe('Frosted Glass Promo', () => {
     expect(container).toMatchSnapshot();
   });
 
+  // what's the desired behavioir here?
   it('when given props for an a Optimo promo with no image', () => {
     const optimoPromoFixtureNoImage = makeOptimoPromoFixture({});
```

Comment:
This snapshot also changes. However what's the desired behaviour here?

Reply:
I believe it's ok it's updated since this would be the default Image behaviour now. It looks like we pass approproate Image props in for this component.